pkg/operator/defaultscccontroller/defaultscccontroller.go

```diff
@@ -150,7 +150,7 @@ func (c *DefaultSCCController) Sync(key types.NamespacedName) error {
 	}
 
 	if len(mutated) > 0 {
-		klog.Infof("[%s] default scc has been mutated %s", ControllerName, mutated)
+		klog.Infof("[%s] default scc has been mutated %s, reset the changes by a) delete the mutated SCC object(s) using oc b) delete any of the `apiserver-*` Pods from the `openshift-apiserver` namespace", ControllerName, mutated)
```

Comment:
do we need b) ?

How about: has been mutated. Revert change by deleting the mutated SCCs and they will be recreated automatically.

Reply:
@sttts  Yes, I think so, we need to restart the openshift-apiserver which will create the missing SCC in a post-startup hook.

Reply:
was reading this as "a or b". It's fine.

Reply:
> /hold
> 
> you need to describe how they actually grant the permissions too, or a pointer to docs.

I don't think we can fit it in a message. All official documentation on SCC dates back to 3.9 and it basically asks to add user/group to the default SCC. 

I opened a BZ to fix the doc - https://bugzilla.redhat.com/show_bug.cgi?id=1822324